models/action.go

```diff
@@ -725,26 +729,28 @@ func GetFeeds(opts GetFeedsOptions) ([]*Action, error) {
 		if repoIDs, err = env.RepoIDs(1, opts.RequestedUser.NumRepos); err != nil {
 			return nil, fmt.Errorf("GetUserRepositories: %v", err)
 		}
+
+		cond = cond.And(builder.In("repo_id", repoIDs))
+	}
+
+	if opts.Collaborate {
+		cond = builder.Eq{"user_id": opts.RequestedUser.ID}.Or(
+			builder.Expr(`repo_id IN (SELECT repo_id FROM "access" WHERE access.user_id = ?)`, opts.RequestedUser.ID))
+	} else {
+		cond = builder.Eq{"user_id": opts.RequestedUser.ID}
```

Comment:
This is not right. The assignments on lines 737 and 740 will overwrite the assignment on line 733.